client/components/QuestionsAndAnswers/ModalForm.jsx

```diff
@@ -1,51 +1,238 @@
+        <InputArea
+          id="email"
+          type="email"
+          maxlength="60"
+          placeholder="Why did you like the product or not?"
+          onChange={(e) => setNewQuestion({ ...newQuestion, email: e.target.value })}
+        />
+        <Notes>
+          For authentication reasons, you will not be emailed
+        </Notes>
+      </Container1>
+      <SubmitInput
+        type="submit"
+        value="Submit Question"
+        onClick={handleSumbit}
       />
-      <Notes>
-        For privacy reasons, do not use your full name or email address
-      </Notes>
-    </Container1>
-    <Container1>
-      <LabelArea htmlFor="email">
-        Your Email
-      </LabelArea>
-      <InputArea
-        id="email"
-        type="email"
-        maxlength="60"
-        placeholder="Why did you like the product or not?"
+    </form>
+  );
+};
+
+export const AnswerForm = ({
+  newAnswer,
+  setNewAnswer,
+  setIsAdd,
+  handleAddAnswer,
+}) => {
+  const handleAnswerSumbit = (e) => {
+    e.preventDefault();
+    handleAddAnswer(newAnswer);
+    setIsAdd(false);
+  };
+
+  return (
+    <form>
+      <Container1>
+        <LabelArea htmlFor="yourAnswer">
+          Your Answer
+        </LabelArea>
+        <LargeText
+          id="yourAnswer"
+          type="text"
+          maxlength="1000"
+          onChange={(e) => setNewAnswer({ ...newAnswer, body: e.target.value })}
+        />
+      </Container1>
+      <Container1>
+        <LabelArea htmlFor="yourNickname">
+          What is your nickname
+        </LabelArea>
+        <InputArea
+          id="yourNickname"
+          type="text"
+          maxlength="60"
+          placeholder="Example: jack543!"
+          onChange={(e) => setNewAnswer({ ...newAnswer, answerer_name: e.target.value })}
+        />
+        <Notes>
+          For privacy reasons, do not use your full name or email address
+        </Notes>
+      </Container1>
+      <Container1>
+        <LabelArea htmlFor="yourEmail">
+          Your Email
+        </LabelArea>
+        <InputArea
+          id="yourEmail"
+          type="email"
+          maxlength="60"
+          placeholder="Example: jack@email.com"
+          onChange={(e) => setNewAnswer({ ...newAnswer, email: e.target.value })}
+        />
+        <Notes>
+          For authentication reasons, you will not be emailed
+        </Notes>
+      </Container1>
+      <Container1>
+        <LabelArea htmlFor="photos">
+          Upload your photos
+        </LabelArea>
+        <InputArea
+          id="photos"
+          type="file"
+          accept="image/*"
+          multiple
+        />
+      </Container1>
+      <SubmitInput
+        type="submit"
+        value="Submit Answer"
+        onClick={handleAnswerSumbit}
       />
-      <Notes>
-        For authentication reasons, you will not be emailed
-      </Notes>
-    </Container1>
-    <SubmitInput type="submit" value="Submit Question" />
-  </form>
-);
+    </form>
+  );
+};
```

Comment:
Consider breaking this component into subcomponents for clarity/readability. It looks like you could break this down into AnswerForm, QuestionForm, AnswerModal, and Question Modal.

Reply:
haha, for sure, it's getting pretty mess